models/custom-view/src/custom-view-permission/presets/manage-only-permissions.ts

```diff
@@ -0,0 +1,26 @@
+import { entryPointUriPathToResourceAccesses } from '@commercetools-frontend/application-config/ssr';
+import { CUSTOM_VIEW_HOST_ENTRY_POINT_URI_PATH } from '@commercetools-frontend/constants';
+import camelCase from 'lodash/camelCase';
+import sampleSize from 'lodash/sampleSize';
+import upperFirst from 'lodash/upperFirst';
+import CustomApplicationPermission from '../builder';
+import { supportedManageOAuthScopes } from '../constants';
+
+const preset = (additionalPermission = '') => {
+  const resourceAccesses = entryPointUriPathToResourceAccesses(
+    CUSTOM_VIEW_HOST_ENTRY_POINT_URI_PATH,
+    [additionalPermission]
+  );
```

Comment:
ℹ️ This requires the change in App Kit to be released: https://github.com/commercetools/merchant-center-application-kit/pull/3272